test/unit/math/fwd/scal/fun/log_inv_logit_diff_test.cpp

```diff
@@ -0,0 +1,38 @@
+#include <stan/math/fwd/scal.hpp>
```

Comment:
Thanks for writing all these tests out.  I think going forward we're going to be OK with just the rev and fwd tests without the additional mix tests.  Please leave them in now that you did them.  The whole thing is going to have to get replaced with an autodiff test framework.